src/transformers/modeling_outputs.py

```diff
@@ -841,3 +882,31 @@ class SemanticSegmentationModelOutput(ModelOutput):
     logits: torch.FloatTensor = None
     hidden_states: Optional[Tuple[torch.FloatTensor]] = None
     attentions: Optional[Tuple[torch.FloatTensor]] = None
+
+
+@dataclass
+class ImageClassificationModelOutput(ModelOutput):
```

Comment:
```suggestion
class ImageClassifierOutput(ModelOutput):
```
Similar to `SequenceClassifierOutput`/`TokenClassifierOutput`?

Reply:
More similar to `ImageSegmentationModelOutput`

Reply:
I think `ImageClassifierOutput` would be more consistent indeed.

Reply:
Then should we also change `ImageSegmentationModelOutput` to `ImageSegmentatorOutput`?

Reply:
Yes.